### PR TITLE
NSP: Add references. Add parameters.

### PR DESCRIPTION
**Pull request by odinnordico**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

Changes in this PR are for TAP 1.6

### CODE REVIEW

File: namespace-provisioner/issues-limitations.hbs.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Namespace Provisioner reference
+
+## Known limitations and issues
+
+If you are using a GitOps repository to manage the list of namespaces, all the namespaces in the list must exist in the cluster. The provisioner application fails to reconcile if the namespaces do not exist on the cluster.
+
+To use different private repositories, the secret used for each entry (gitops_install, additional_sources) must be a unique name. Re-using the same secret is not supported due to a limitation in kapp-controller.
+
+he namespace provisioner adds the finalizer `namespace-provisioner.apps.tanzu.vmware.com/finalizer` to the managed namespace. If you are in controller mode and want to uninstall the namespace provisioner or want to change the to GitOps mode (see [Manage a list of developer namespaces](provision-developer-ns.hbs.md)), then for all the namespaces managed by the namespace provisioner, the finalizer musth be manually cleaned before perform any of those operations by removing the label (see (provision-developer-ns.hbs.md#manage-list)) or manually editing the namespace manifest
```
Comment (atmandhol) [inline namespace-provisioner/issues-limitations.hbs.md:9]: Missing `T` in the in Line 9

Comment (odinnordico) [reply namespace-provisioner/issues-limitations.hbs.md]: Rephrase the paragraph

File: namespace-provisioner/reference.hbs.md [hunk 2]
```diff
@@ -1,29 +1,7 @@
 # Namespace Provisioner reference
 
-This topic contains know issues and limitations
-## Known limitations and issues
+This topic contains know issues and limitations, the default resources and some technical details
```
Comment (atmandhol) [inline namespace-provisioner/reference.hbs.md:3]: `typo`: Known Issue

Comment (atmandhol) [reply namespace-provisioner/reference.hbs.md:3]: `and some technical details` is too generic

**Review by atmandhol — APPROVED** [review]

File: namespace-provisioner/reference.hbs.md (hunk 2 above)
Comment (odinnordico) [reply namespace-provisioner/reference.hbs.md]: Rephrase the paragraph

**Review by atmandhol — APPROVED** [review]

File: namespace-provisioner/about.hbs.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Controller mode
     ![Diagram of Namespace Provisioner controller.](../images/namespace-provisioner-controller.png)
 
     - The list of developer namespaces is managed by the Namespace Provisioner controller using a label selector `apps.tanzu.vmware.com/tap-ns=""`
-    - Namespace Provisioner creates [default resources](reference.md#default-resources) in all managed namespaces.
+    - Namespace Provisioner creates [default resources](default-resources.hbs.md) in all managed namespaces.
     - Namespace Provisioner creates additional Platform Operator templated resources stored in Git repository locations specified under the `additional_sources` section in Namespace Provisioner configuration. For more information, see [Customize Installation of Namespace Provisioner](customize-installation.md).
```
Comment (shaheerkootteeri) [inline namespace-provisioner/about.hbs.md:23]: Is this customize-installation.hbs.md ?

Comment (shaheerkootteeri) [reply namespace-provisioner/about.hbs.md:23]: As discussed with @kcoriordan , these changes in file extensions will be handled in a separate PR . Closing this comment

Comment (shaheerkootteeri) [inline namespace-provisioner/about.hbs.md:23]: Is this customize-installation.hbs.md ?

File: namespace-provisioner/issues-limitations.hbs.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Namespace Provisioner reference
+
+## Known limitations and issues
+
+- If you are using a GitOps repository to manage the list of namespaces, all the namespaces in the list must exist in the cluster. The provisioner application fails to reconcile if the namespaces do not exist on the cluster.
+
+- To use different private repositories, the secret used for each entry (gitops_install, additional_sources) must be a unique name. Re-using the same secret is not supported due to a limitation in kapp-controller.
+
+- Before performing any of these operations, such as uninstalling the provisioner or transitioning from the Controller mode to the GitOps mode, ensure that you clean up the finalizer (`namespace-provisioner.apps.tanzu.vmware.com/finalizer`) for all namespaces managed by the provisioner. This can be done either by removing the label used to set the namespaces as managed (refer to[Manage a list of developer namespaces](provision-developer-ns.hbs.md)), or by manually editing the namespace manifest and removing the finalizer entry.
```
Comment (kcoriordan) [inline namespace-provisioner/issues-limitations.hbs.md:9]: Is '..all namespaces managed by the provisioner...' referring to Namespace Provisioner or the provisioner application?

[the remark this answers is not in the document]
Comment (odinnordico) [reply namespace-provisioner/issues-limitations.hbs.md:9]: It refers to the Namespace provisioner

Comment (odinnordico) [reply namespace-provisioner/issues-limitations.hbs.md:9]: It refers to the Namespace Provisioner

File: namespace-provisioner/parameters.hbs.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
+# Namespace Provisioner reference
```
[the remark this answers is not in the document]
Comment (odinnordico) [reply namespace-provisioner/parameters.hbs.md:1]: That is the correct location, thank you

**Pull request merged**